### ISSUE TITLE
Introduce serving proto for identifier service [phab:D1888]

### ISSUE THREAD

**Issue by kythe** [google/kythe#2330]
_Created by **djrenren** at 2017-08-17 21:26:03_: 

Adds the necessary proto for the internals of the identifier service

**Comment by kythe** [comment]
_**zarko** accepted this revision at 2017-08-17 21:31:43_

**Comment by kythe** [comment]
_**schroederc** commented inline (context lost during migration) at 2017-08-17 22:24:57_:

Based on our offline conversation, I envisioned something like this:

```
message IdentifierMatch {
  message Node {
    string ticket = 1;
    string node_kind = 2;
    string node_subkind = 3;
  }

  string qualified_name = 1;
  string base_name = 2;
  repeated Node node = 3;
}
```

There may be multiple nodes associated per qualified name.

**Comment by kythe** [comment]
_**djrenren** wrote at 2017-08-21 20:41:16_:

  - Allow multiple Nodes to to match a singular qualified name

**Comment by kythe** [comment]
_**djrenren** closed via commit 020a5de9cd4fb1e348c52bf37f54cfbd2c8f6c2a at 2017-08-21 20:42:28_
